### PR TITLE
#419 Updated the material account in side bar

### PR DESCRIPTION
**Pull request by VigneshPalanisamy**

### CODE REVIEW

File: app/src/main/java/org/worshipsongs/activity/NavigationDrawerActivity.java [hunk 1]
```diff
@@ -49,9 +56,33 @@ public void init(Bundle bundle)
...
+    private MaterialSectionListener getUpdateDbIntent()
     {
-        return new Intent(NavigationDrawerActivity.this, UpdateSongsDatabaseActivity.class);
+        return new MaterialSectionListener()
+        {
+            @Override
+            public void onClick(MaterialSection materialSection)
+            {
+                Intent updateSongs = new Intent(NavigationDrawerActivity.this, UpdateSongsDatabaseActivity.class);
+                startActivityForResult(updateSongs, 555);
```
Comment (madasamys) [inline app/src/main/java/org/worshipsongs/activity/NavigationDrawerActivity.java]: Define constant

File: app/src/main/java/org/worshipsongs/activity/NavigationDrawerActivity.java [hunk 2]
```diff
@@ -49,9 +56,33 @@ public void init(Bundle bundle)
...
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data)
+    {
+        switch (requestCode) {
+            case 555:
+                long noOfSongs = sharedPreferences.getLong(CommonConstants.NO_OF_SONGS, 0);
+                if(this.getAccountList().size() > 0) {
+                    this.getAccountAtCurrentPosition(0).setSubTitle(noOfSongs +" Songs are available");
```
Comment (madasamys) [inline app/src/main/java/org/worshipsongs/activity/NavigationDrawerActivity.java]: Get strings value from resource file

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -126,5 +126,5 @@
     <string name="storage_permission_meaage">Tamil christian Worshipsongs doesn\'t have permission to write file on storage. Tap on settings to enable storage permission</string>
     <string name="favourite_title">Add favourite</string>
     <string name="font_size">Font size</string>
-
+    <string name="noOfSongsAvailable">%1$d Songs are available</string>
```
Comment (madasamys) [inline app/src/main/res/values/strings.xml]: Add this to tamil localization file too.

Comment (VigneshPalanisamy) [inline app/src/main/res/values/strings.xml]: added

**Pull request merged**